controllers/articles.controller.js

```diff
@@ -20,3 +21,27 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+// exports.updateVotes = (req, res, next) => {
+//   const newVotes = req.body;
+//   const article_id = req.params.article_id;
+//   changeVotes(newVotes, article_id)
+//     .then((article) => {
+//       return res.status(200).send(article);
+//     })
+//     .catch(next);
+// };
+
+exports.updateVotes = (req, res, next) => {
+  const newVotes = req.body;
+  const article_id = req.params.article_id;
+  const promises = [changeVotes(newVotes, article_id)];
+  if (article_id) {
+    promises.push(checkArticleExists(article_id));
+  }
+  Promise.all(promises)
+    .then((article) => {
+      return res.status(200).send(article);
```

Comment:
You should send article on an object - change this to {article} 

Then in your test, you can access the information on body.article.